airflow/serialization/serialized_objects.py

```diff
@@ -1081,12 +1081,12 @@ def deserialize_dag(cls, encoded_dag: Dict[str, Any]) -> 'SerializedDAG':
                 setattr(task.subdag, 'parent_dag', dag)
 
             if isinstance(task, MappedOperator):
-                for d in (task.mapped_kwargs, task.partial_kwargs):
-                    for k, v in d.items():
-                        if not isinstance(v, _XComRef):
-                            continue
+                expansion_kwargs = task._get_expansion_kwargs()
```

Comment:
This was a bug that I don't know how we didn't hit before? The serialized task as a result had these objects left as _XcomRef, which as it is a named tuple has a `__len__` of 2 😱

Reply:
Because those attributes _shouldn’t_ be accessed by the scheduler anyway? So this likely only caused the task to be rendered incorrectly in UI.